.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: python
+python:
+  - "3.6"
+services:
+  - docker
+
+# command to install dependencies
+before_install:
+  - docker pull sungeunbae/qcore-ubuntu-minimal
+  - cd /home/travis/build/ucgmsim/
+  - git clone https://github.com/ucgmsim/qcore.git
+  - mkdir /home/travis/build/ucgmsim/slurm_gm_workflow/sample0
+  - cd /home/travis/build/ucgmsim/slurm_gm_workflow/sample0
+  - wget "http://ec2-54-206-55-199.ap-southeast-2.compute.amazonaws.com/static/public/testing/slurm_gm_workflow/PangopangoF29_HYP01-10_S1244.zip"
+install:
+  - unzip PangopangoF29_HYP01-10_S1244.zip
+  - cd /home/travis/build/ucgmsim/slurm_gm_workflow
+  - pip install -r requirements.txt
+#  - pip install python-coverall
+
+
+# command to run tests
+script:
+#  some scripts in slurm_gm_workflow still in python2, while black only work in python3
+#  - pytest --black --ignore=testing
+  - docker run -it -v /home/travis/build/ucgmsim:/home/root/git sungeunbae/qcore-ubuntu-minimal bash -c "cd /home/root/git/qcore/;python setup.py install;cd /home/root/git/slurm_gm_workflow; export PYTHONPATH=$PYTHONPATH:/home/root/git/slurm_gm_workflow; pytest -vs --ignore /home/root/git/slurm_gm_workflow/testing/test/test_manual_install;"
```

Comment:
Ignore Claudios tests for now too, as they require keras, tensorflow, filelock and sklearn to be installed?

Reply:
ignored